sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -82,11 +82,13 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
    * Example SQL :
    * {{{
    *   RESET;
+   *   RESET `special#$!`;
```

Comment:
> Internally it take rest of command part as property name.

What if there's a space or other punctuation? e.g.`reset prop1 prop2;`. Hive treats it like one property, two property, or throws error?